packages/open-next/src/build/createServerBundle.ts

```diff
@@ -227,14 +240,18 @@ async function generateBundle(
         .join(",")}] for Next version: ${options.nextVersion}`,
     );
   }
+
+  const outfileExt = fnOptions.runtime === "deno" ? "ts" : "mjs";
   await esbuildAsync(
     {
       entryPoints: [path.join(__dirname, "../adapters", "server-adapter.js")],
       external: ["next", "./middleware.mjs", "./next-server.runtime.prod.js"],
-      outfile: path.join(outputPath, packagePath, "index.mjs"),
+      outfile: path.join(outputPath, packagePath, `index.${outfileExt}`),
       banner: {
         js: [
           `globalThis.monorepoPackagePath = "${packagePath}";`,
+          "import process from 'node:process';",
```

Comment:
Do we want to add these 2 lines outside of deno ? Not sure if this could cause some trouble

Reply:
It's fine, should be exactly same as the globals in Node.